RpbInCollectionImprovement

TobiasNx opened the pull request:
@dr0i should we use a local map/ttl instead of the dependencies of github for the RPB SKOS lookups? And replicate/mirror these files locally?

https://github.com/hbz/lobid-resources/blob/ee1a85c59346503704567f711ffa9616db795775/src/main/resources/alma/fix/maps.fix

TobiasNx commented:
we already do this with other vocabs from the lobid vocabs. grep the new file daily keep the old file as fall back

dr0i commented:
We can do that.  If I understand that correctly, we need to cron-get these files:
- https://raw.githubusercontent.com/lobid/vocabs/master/rpb/rpb2.ttl
- https://raw.githubusercontent.com/hbz/lobid-vocabs/master/rpb/rpb-spatial.ttl
- https://raw.githubusercontent.com/hbz/lobid-vocabs/master/rpb/rpb.ttl

say at Friday 20:00 o'clock ?

TobiasNx commented:
@dr0i I added the files locally to where you can cron-get these.

dr0i commented:
>  @dr0i I added the files locally to where you can cron-get these.

I don't understand. E.g. where do I get `picaCreatorId2Isil.tsv` from?

TobiasNx commented:
> > @dr0i I added the files locally to where you can cron-get these.
> 
> I don't understand. E.g. where do I get `picaCreatorId2Isil.tsv` from?

sorry for the confusion I just fixed the formatting. You do not need this file.

TobiasNx commented:
> > We can do that.  If I understand that correctly, we need to cron-get these files:
> > 
> > * https://raw.githubusercontent.com/lobid/vocabs/master/rpb/rpb2.ttl
> > * https://raw.githubusercontent.com/hbz/lobid-vocabs/master/rpb/rpb-spatial.ttl
> > * https://raw.githubusercontent.com/hbz/lobid-vocabs/master/rpb/rpb.ttl
> > 
> > say at Friday 20:00 o'clock ?
> 
> Why not `git pull origin master` the whole repo? Just thinking this might be the cleaner approach...

I would be fine with that. But then we need to redo some stuff.

dr0i commented:
> Why not git pull origin master the whole repo? Just thinking this might be the cleaner approach... 

Why would this be cleaner? If you would do this relative to this repo  you would have that repo in the repo, which would suggest to add it to `.gitignore`. Doing it outside this repo would mean some hassle to reference the resources. Anyway, the best way to store/get these files would be in `.../maps/...` in the resources directory of this' repo.

acka47 commented:
I am convinced, go ahead as you have planned originally.

dr0i commented:
Added downloading to cron. Commited cron into internal repo. Closing.
